lessons/pydata/eda-univariate-timeseries/index.ipynb

```diff
@@ -0,0 +1,4745 @@
+{
```

Comment:
<p>Chybějící hodnoty se mohou</p><p> </p><p>samy/i (není to jen pro ladies)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>